src/redismodule.h

```diff
@@ -357,6 +357,7 @@
 #define RedisModule_SetModuleAttribs ValkeyModule_SetModuleAttribs
 #define RedisModule_IsModuleNameBusy ValkeyModule_IsModuleNameBusy
 #define RedisModule_WrongArity ValkeyModule_WrongArity
+#define RedisModule_UpdateRuntimeArgs ValkeyModule_UpdateRuntimeArgs
```

Comment:
The Redis module API is just for backward compatibilty. We don't add functions to that API.

```suggestion
```